Modules/WorldBuffModule.cs

```diff
@@ -128,7 +128,7 @@ public class WorldBuffModule : ModuleBase<SocketCommandContext>
                 if(buff != null){
                     context.Remove(buff);
                     await context.SaveChangesAsync();
-                    await Context.Message.AddReactionAsync(Emote.Parse(_icons[buff.Type]));
+                    // await Context.Message.AddReactionAsync(Emote.Parse(_icons[buff.Type]));
```

Comment:
This can be uncommented, commited by accident and commented it out since it doesn't work on my local.